environment.yml

```diff
@@ -4,14 +4,14 @@ channels:
   - defaults
 
 dependencies:
-  - ale>=0.3.2
+  - ale=0.5.0
```

Comment:
Can we make this a greater than or does it need to be 0.5.0?

Reply:
0.6 introduced an API breaking change and the current version of ALE that was used for ISIS is based on 0.5.0